Cab share/Login/LoginView.swift

```diff
@@ -6,12 +6,27 @@
 //
 
 import SwiftUI
+import FirebaseCore
+import FirebaseAuth
 
 struct LoginView: View {
     @State private var email = ""
     @State private var password = ""
-    @State private var signIn = false
+    @State private var signInView = false
+    @State private var userIsLoggedIn = false
+    @State private var userSignedUp = false
+    @State private var userSignedIn = false
     var body: some View {
+        if userSignedIn {
+            MainView()
+        } else if userSignedUp {
+            UserView()
+        } else {
+            content
```

Comment:
What content is in here? I didn't get it.

Reply:
content takes the view below